securedrop/debian/app-code/etc/apparmor.d/usr.sbin.tor

```diff
@@ -49,7 +49,7 @@
   /var/lib/tor/services/client_keys.tmp rw,
   /var/lib/tor/services/hostname w,
   /var/lib/tor/services/hostname.tmp rw,
-  /var/lib/tor/sevices/private_key rw,
+  /var/lib/tor/services/private_key rw,
```

Comment:
scary

Reply:
I think those entries are stale leftover from previous releases and we shall rebuild the profile. I do not think that file exists or is supposed to exist anyway. Relevant keys are in `sourcev3` and `journalistv3`.